### DISCUSSION TITLE
Prometheus模块是否考虑支持自定义信息到metrics接口内？

### DISCUSSION THREAD

**Discussion by helloHT** [midwayjs/midway#3367] [Q&A] [answered]
### 🚀 The feature, motivation（功能、动机）

运维有个系统监控Prometheus，业务系统有个指标需要通过Prometheus提供给运维，但是目前没看到可以自定义信息到metrics中。

**Comment by czy88840616** [comment, marked as answer]
参考 readme 进行自定义：https://github.com/midwayjs/midway/tree/main/packages/prometheus
